web_tool_script_1.R

```diff
@@ -57,22 +57,29 @@ if (new_data == TRUE) {
     company_emissions
   )
 } else {
-  currencies <- read_file(file.path(file_location, "currencies.fst"))
+  currencies <- fst::read_fst(file.path(file_location, "currencies.fst"))
 
-  fund_data <- read_file(file.path(file_location, "fund_data.fst"))
+  read_fst_or_return_null <- function(fst_file) {
+    if (!file.exists(fst_file)) {
+      return(NULL)
+    }
 
-  fin_data <- read_file(file.path(file_location, "fin_data.fst"))
+    fst::read_fst(fst_file)
+  }
+
+  fund_data_path <- file.path(file_location, "fund_data.fst")
+  fund_data <- read_fst_or_return_null(fund_data_path)
```

Comment:
I need eyes here. 

In general it is a bad idea for a function to be type-unstable -- sometimes returning one thing (e.g. a dataframe) and sometimes another thing (e.g. `NULL`). But I tolerate this here in the interest of keeping the code running. If instead of `NULL` I throw an error, I brake stuff -- and for now there are other priorities.

Reply:
I agree, in both senses.... it's probably not a great idea, but until broader changes are made this is the expected behavior.